### ISSUE TITLE
Does it support the SSL related function？

### ISSUE THREAD

**Issue by wangke0809** [intel/intel-sgx-ssl#25]
Hi,

Does it support the SSL related function？Such as `SSL_new`, `SSL_read`, `SSL_write`, etc.

When I use these functions, it tells me `Undefined reference` .

**Comment by naishai** [comment]
no, currently the SSL library is not supported (it's not even built...)
